lxd/instance/drivers/driver_lxc.go

```diff
@@ -5993,7 +6035,7 @@ func (d *lxc) insertMountLXD(source, target, fstype string, flags int, mntnsPID
 		fmt.Sprintf("%d", pidFdNr),
 		mntsrc,
 		target,
-		fmt.Sprintf("%v", shiftfs),
+		fmt.Sprintf("%s", idmapType),
```

Comment:
Not sure of the benefits of each but you could do a type conversion using `string(idmapType)` here rather than have the function call to `fmt.Sprintf()`. There's a few other places you use fmt.Sprintf(), but not sure if there is a specific reason you do that?